docs/app/Examples/Button/Basic.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
+import {Button} from 'stardust';
+
+export default class extends Component {
+  render() {
+    return (
+      <div>
+        <Button className = 'basic'>Standard</Button>
+        <Button className = 'red basic'>Red</Button>
+        <Button className = 'orange basic'>Orange</Button>
+        <Button className = 'yellow basic'>Yellow</Button>
+        <Button className = 'olive basic'>Olive</Button>
+        <Button className = 'green basic'>Green</Button>
+        <Button className = 'teal basic'>Teal</Button>
+        <Button className = 'blue basic'>Blue</Button>
+        <Button className = 'violet basic'>Violet</Button>
+        <Button className = 'purple basic'>Purple</Button>
+        <Button className = 'pink basic'>Pink</Button>
+        <Button className = 'brown basic'>Brown</Button>
+        <Button className = 'grey basic'>Grey</Button>
+        <Button className = 'black basic'>Black</Button>
```

Comment:
Hm, I would think the linter should have caught this.  There are extra spaces around the `=` operator here.